frontend/src/views/Home.vue

```diff
@@ -1,8 +1,45 @@
 <template>
-  <div class="main-content home">
-    <AppLogo :first="'Smart'" :last="'Course'"/>
-    <p>Making the most of your degree.</p>
-    <Search/>
+  <div>
+    <section class="main-content home">
```

Comment:
this page lookin' good also

Reply:
Definitely a WIP. Need to play with mobile sizing, also not certain on card design for info panels.